src/e2e-test/features/bigquery/source/BigQueryToGCS.feature

```diff
@@ -116,3 +116,41 @@ Feature: BigQuery source - Verification of BigQuery to GCS successful data trans
     Then Open and capture logs
     Then Verify the pipeline status is "Succeeded"
     Then Verify data is transferred to target GCS bucket
+
+  @BQ_SOURCE_DATATYPE_TEST @GCS_SINK_MULTI_PART_UPLOAD
+Scenario:Validate successful records transfer from BigQuery to GCS with bucket having delete multi part upload policy enabled
+  Given Open Datafusion Project to configure pipeline
+  When Expand Plugin group in the LHS plugins list: "Source"
+  When Select plugin: "BigQuery" from the plugins list as: "Source"
+  When Expand Plugin group in the LHS plugins list: "Sink"
+  When Select plugin: "GCS" from the plugins list as: "Sink"
+  Then Open BigQuery source properties
+  Then Enter BigQuery property reference name
+  Then Enter BigQuery property projectId "projectId"
+  Then Enter BigQuery property datasetProjectId "projectId"
+  Then Override Service account details if set in environment variables
+  Then Enter BigQuery property dataset "dataset"
+  Then Enter BigQuery source property table name
```

Comment:
Use new framework steps, for entering all the field properties. Check the new tests added for BQ or GCS.

Reply:
Done , changes added as per new framework.